src/locales/en.json

```diff
@@ -1,14 +1,14 @@
 {
   "nav": {
-    "item_files": "Files",
+    "item_files": "Drive",
     "item_recent": "Recent",
     "item_shared": "Shared by me",
     "item_activity": "Activity",
     "item_trash": "Trash",
     "item_settings": "Settings"
   },
   "breadcrumb": {
-    "title_files": "Files",
+    "title_files": "Drive",
```

Comment:
We chose to go for "Drive" here to match what is done by Google, but @pierrevdk will have the last word anyway